app/assets/javascripts/hydra-editor/keyboard_support.js

```diff
@@ -1,14 +0,0 @@
-Blacklight.onLoad(function() {
-  $('body').on('keypress', '.multi-text-field', function(event) {
-    var $activeField = $(event.target).parents('.field-wrapper'),
-        $activeFieldControls = $activeField.children('.field-controls'),
-        $addControl = $activeFieldControls.children('.add'),
-        $removeControl = $activeFieldControls.children('.remove');
-    if (event.keyCode == 13) {
-      event.preventDefault();
-      $addControl.click()
-      $removeControl.click()
-    }
-  });
-});
-
```

Comment:
FYI, I have been tabbing to the button and hitting enter, as opposed to using the mouse.  The button is always one tab away from the input field, so it's pretty easy to get to.  The submit button was the tough one to navigate to since our forms are currently very big (and most fields are optional).

I'm not a keyboard-only user, mind you, I just happen to avoid the mouse as much as I can due to wrist issues.  Leaving the hands in one spot is just easier.